src/nikhilMJun20/SumOfNumbersInAString.java

```diff
@@ -0,0 +1,26 @@
+/*Assignment 32 : 21st July 2020
+WAP to do sum of numbers in given statement. (Use split method and Exception handling concept. Do not use character class methods and ascii concept.)
+
+Input : I have 5 Years 6 month of experience.
+Output : 11
+*/
+
+package nikhilMJun20;
+
+public class SumOfNumbersInAString {
+	double getSumOfNumbers (String str) {
+		String[] strArray = str.split(" ");
+		double sum = 0.0;
+		for(int index = 0; index<strArray.length; index++) {
+			try {
+				sum += Double.parseDouble(strArray[index]);
```

Comment:
I am happy that you directly use it. Good job.

Reply:
Thank you, @KrishnaKTechnocredits !